specs/core/0_beacon-chain.md

```diff
@@ -397,6 +399,15 @@ The types are defined topologically to aid in facilitating an executable version
 }
 ```
 
+#### `ActiveRegistry`
+
+```python
+    [{
```

Comment:
I think this should be a container type with a single list field just so we can easily talk about the type